build.gradle.kts

```diff
@@ -158,7 +158,11 @@ project(":plugin") {
     version = "0.2.$patchVersion.${prop("buildNumber")}$versionSuffix"
     intellij {
         pluginName = "intellij-rust"
-        val plugins = mutableListOf(project(":intellij-toml"), "IntelliLang")
+        val plugins = mutableListOf(
+            project(":intellij-toml"),
+            "IntelliLang",
+            "PsiViewer:$platformVersion-SNAPSHOT"
```

Comment:
Let's avoid relying on some accident relation between platform and plugin versions.
For example, the latest version for 191 is `191.4212` but not `191-SNAPSHOT`
I suggest adding `psiViewerPluginVersion` property to `gradle-%platformVersion%.properties` and use `prop("psiViewerPluginVersion")`

Reply:
Done.